ftests/nuxeo-server-tests/pom.xml

```diff
@@ -80,6 +80,14 @@
       <artifactId>nuxeo-platform-oauth1</artifactId>
       <scope>test</scope>
     </dependency>
+
+    <!-- excluded from chemistry-opencmis-commons-impl for groupId & version issues -->
+    <dependency>
+      <groupId>com.fasterxml.woodstox</groupId>
+      <artifactId>woodstox-core</artifactId>
+      <scope>test</scope>
+    </dependency>
+
```

Comment:
Added this dependency to fix some failing test org.nuxeo.ftest.server.ITOAuth2Test(nuxeo-server-tests).testAuthorizationOnCMIS:
https://qa.nuxeo.org/jenkins/view/TestAndPush/job/TestAndPush/job/ondemand-testandpush-nalkotob/146/testReport/junit/org.nuxeo.ftest.server/ITOAuth2Test(nuxeo-server-tests)/testAuthorizationOnCMIS/